src/App.vue

```diff
@@ -1,15 +1,12 @@
 <template>
-  <img alt="Vue logo" src="./assets/logo.png">
```

Comment:
if logo is not necessary anymore, plaese delete that asset as well.

Reply:
sure